package.json

```diff
@@ -57,6 +57,7 @@
     "@nodesecure/js-x-ray": "^6.0.1",
     "@nodesecure/vuln": "^1.7.0",
     "@slimio/config": "^1.2.0",
+    "lodash": "^4.17.21",
```

Comment:
No, you only need to install `lodash.merge` package (and not the entire lodash package).